tests/summary/dummy_binning_descriptions.py

```diff
@@ -3,6 +3,8 @@
 
 bins_py = {"in": "Jet_Py", "out": "py_leadJet", "bins": dict(edges=[0, 20., 100.], overflow=True), "index": 0}
 
+bins_electron_pT = {"in": "sqrt(Electron_Px**2 + Electron_Py**2)", "out": "electron_pT",
+                    "bins": dict(nbins=200/10., low=0.0+0.0, high=2.0*10**2)}
```

Comment:
Python itself will be resolving the calculations for the binning, before it gets to numexpr.  You could change at least one (maybe not all, though) to be strings.  Keeping one a string and one a float would be good to validate that this still works with normal floats being passed in (which might happen if the YAML config is just given a float / int-like expression).

Reply:
Hmm, you sure? I was under the impression that anything that doesn't match an integer or float  is parsed as a string by pyyaml? So at the moment these are all fed as strings to numexpr, but that doesn't matter in the end as pd.eval can work with both numeric and string literals 

```python
In [7]: pd.eval(2) == pd.eval('2')
Out[7]: True```

Reply:
Sorry if I wasn't very clear.  I guess there were two points in the above comment:

Firstly, `dict(nbins=200/10., low=0.0+0.0, high=2.0*10**2)` is written within python and not YAML, so the values for each key are all parsed by python, meaning they're directly calculated before the dictionary is created.  The dict for `bins` that is passed to `pd.eval` will therefore just be: `dict(nbins=20., low=0.0,  high=200.)`.

The second point was that it's probably good to test both floats and strings being passed to the underlying method.  I already test the raw float version I think, so maybe this isn't actually worth listening.

Reply:
Ahh yeah, sorry I'm being dim and thought I was working with a yaml file...